app/Http/Controllers/Admin/WorkspaceController.php

```diff
@@ -25,9 +26,21 @@ public function __construct(WorkspaceRepository $workspaceRepository)
      */
     public function index()
     {
-        $workspaces = $this->workspace->getWorkspaces();
+        $workspaces = $this->workspace->with(['locations' => function ($query) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/WorkspaceController.php</br>**Line**: 29</br>Each line in an array declaration must end in a comma